src/layers/activations/leaky_relu.cpp

```diff
@@ -84,7 +84,7 @@ template <typename TensorDataType, data_layout Layout, El::Device Device>
 void leaky_relu_layer<TensorDataType, Layout, Device>::bp_compute()
 {
   local_bp<TensorDataType>(this->m_negative_slope,
-                           this->get_local_prev_activations(),
+                           this->get_local_activations(),
```

Comment:
@tbennun @fiedorowicz1 Shouldn't this instigate a change in the header file about the backprop requirements.

Reply:
Yes, I was expecting this PR would be merged before in-place. Fixed now.